app/modules/onboardin/api/user.api.ts

```diff
@@ -0,0 +1,18 @@
+import { AxiosPromise } from "axios";
+import { User } from "../models/User";
+
+export interface IUsersRequestParams {
```

Comment:
It's out of requirements, so you could avoid the implementation of paging and sorting.